tests/phpunit/tests/admin/includesSchema.php

```diff
@@ -177,6 +177,47 @@ public function data_populate_options() {
 		);
 	}
 
+	/**
+	 * Safeguards that deprecated timezone strings set as a default in a translation are handled correctly.
+	 *
+	 * @ticket 56468
+	 */
+	public function test_populate_options_when_locale_uses_deprecated_timezone_string() {
+		global $wpdb;
+
+		// Back up.
+		$orig_options  = $wpdb->options;
+		$wpdb->options = self::$options;
+
+		// Ensure the "default" value for the timezone will be set to a deprecated timezone.
+		add_filter(
+			'gettext_with_context',
+			function( $translation, $text, $context ) {
+				if ( '0' === $text && 'default GMT offset or timezone string' === $context ) {
+					return 'America/Buenos_Aires';
+				}
+
+				return $translation;
+			},
+			10,
+			3
+		);
+
+		// Test.
+		populate_options();
+
+		wp_cache_delete( 'alloptions', 'options' );
+
+		$result = get_option( 'timezone_string' );
```

Comment:
Is there a scenario in which `timezone_string` might _already_ be set to `America/Buenos_Aires` due a faulty test earlier in the class/test suite?

Reply:
Not that I can think off with the resets I added.